openfisca_us_data/datasets/acs/acs.py

```diff
@@ -0,0 +1,73 @@
+from openfisca_us_data.utils import US, dataset
+from openfisca_us_data.datasets.acs.raw_acs import RawACS
+from pandas import DataFrame
+import h5py
+
+
+@dataset
+class ACS:
+    name = "acs"
+    model = US
+
+    def generate(self, year: int) -> None:
```

Comment:
The reason this doesn't take a self argument is because of the `dataset` class decorator, making this a class method. I think it's cleaner to have all methods not have the self argument, because self refers to the instance of the class, and the dataset classes are designed not to be instantiated by just imported.